docs/tutorials/how_to_battle_script_command_macro.md

```diff
@@ -17,7 +17,17 @@ In general, `gBattlescriptCurrInstr` tracks the current battle script position a
 Now, how might we add a custom various command case? Here are the steps. We will use `VARIOUS_SET_SIMPLE_BEAM` as an example.
 ### 1. Add a definition to `include/constants/battle_script_commands.h`.
 
-For example, `#define VARIOUS_SET_SIMPLE_BEAM   39`
+For example:
+```diff
+enum CmdVarious
+{
+    ...
+    VARIOUS_SUCKER_PUNCH_CHECK,
++   VARIOUS_SET_SIMPLE_BEAM,
+    VARIOUS_TRY_ENTRAINMENT,
+    ...
+};
```

Comment:
Should we promote adding variouses instead of callnatives?

Reply:
Couldn't agree more with Alex here

Reply:
Also agree, did this out of habit of updating docs.
I'll change the tutorial to account for this.